### ISSUE TITLE
Upload IAP screenshots?

### ISSUE THREAD

**Issue by jonnyijapan** [fastlane/deliver#211]
Can this thing upload IAP screenshots in a batch? The Application Loader seems to fail at this thing (and everything else). Someone noted that deliver can upload metadata like IAP details... can it?

**Comment by KrauseFx** [comment]
Yes, `deliver` can upload binaries and metadata, but not in app purchases. Sure, you could chain multiple calls of `deliver` to upload more binaries.

**Comment by jonnyijapan** [comment]
I mean the screenshots of the IAP for review. I have 20 for my next app and it's a pain doing them one by one. I vote for such a feature.

2015/05/19 20:25$B!"(BFelix Krause <notifications@github.com> $B$N%a%C%;!<%8(B:

**Comment by KrauseFx** [comment]
No, that's not implemented, sorry.

**Comment by jonnyijapan** [comment]
Just an update to tell that I succeeded to upload IAP screenshots using Application Loader, when I finally got around to submit the app. I was able to drag-and-drop the images into image area for each IAP item inside Application Loader. It uploaded all to iTunes Connect. It works, it just takes a while for the upload to reflect itself unto iTunes Connect. I waited for like 10 minutes before the IAP changed status to "Ready to Submit".

So this feature is not 100% needed at this moment. Still the Application Loader/iTunes Connect flow is kind of slow and could be made better, I suppose.
